exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -312,12 +312,13 @@ drill.exec: {
   memory: {
     operator: {
       max: 20000000000,
-      initial: 10000000
+      initial: 10000000,
+      output_batch_size : 536870912
```

Comment:
See note above.

BTW: If you add a new memory-related param, you can use the methods specifically designed for memory. Then you can say things such as 512K or 2MB. The sort code does this. See also the HOCON documentation.